packages/manager/modules/support/src/support.routing.js

```diff
@@ -0,0 +1,24 @@
+import component from './support.component';
+
+import { state as stateToTickets } from './tickets/tickets.routing';
+
+import { STATE_NAME } from './support.constants';
+
+export const state = {
+  name: STATE_NAME,
+  redirectTo: stateToTickets.name,
+  translations: { format: 'json', value: ['.'] },
```

Comment:
I think you could remove it since you add a lot of `.run(/* @ngTranslationsInject:json ./translations */)` and you refresh the translations `onBefore` transitions (https://github.com/ovh-ux/manager/pull/1215/files#diff-10edda547bcaf6534e1f95d95717342eR30)